codetransformer/tests/test_code.py

```diff
@@ -0,0 +1,13 @@
+from codetransformer.code import Code
+
+
+def test_lnotab_roundtrip():
+    def f():  # pragma: no cover
+        a = 1
+        b = 2
+        c = 3
+        d = 4
+        a, b, c, d
+
+    code = Code.from_pycode(f.__code__)
```

Comment:
Should we test that the Code lnotab representation is the dict from line <-> instruction?